vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_util.cc

```diff
@@ -187,9 +187,11 @@ bool one_of(T&& t, Ts&&... ts) {
   return match;
 }
 
-void LogWalletStatusChange(LedgerImpl* ledger,
-                           absl::optional<type::WalletStatus> from,
-                           type::WalletStatus to) {
+bool show_wallet_new_verified = true;
+
+void WalletStatusChange(LedgerImpl* ledger,
```

Comment:
Maybe this should be called `OnWalletStatusChange`?

Reply:
I was thinking about that, too. `OnWalletStatusChange` would imply that this is a callback function, though. If we're OK with that, it's fine with me, too. 👍🏻

Reply:
Yeah, I had the same hesitation. I think it would make sense in this particular scenario, but open to other names too!

Reply:
Yeah, I think so, too.